Ticketing.Client/Controllers/EventsController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Net;
+using System.Web.Http;
+using Ticketing.Db.DAL;
+using Ticketing.Db.Models;
+
+namespace Ticketing.Api.Client.Controllers
+{
+    [Route("/api/venues")]
+    public class EventsController : ApiController
+    {
+        private readonly Repository<Event> _eventRepository;
+        private readonly Repository<Offer> _offerRepository;
+
+        public EventsController(Repository<Event> eventRepository, Repository<Offer> offerRepository)
+        {
+            _eventRepository = eventRepository;
+            _offerRepository = offerRepository;
+        }
+
+        [HttpGet]
+        public async Task<HttpResponseMessage> GetEvents()
```

Comment:
Regarding best practices, async methods should have an 'Async' ending in the naming. Please fix it across whole solution.